README.md

```diff
@@ -146,12 +151,14 @@ Feel free to check the [issues page](../../issues/).
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-I would like to thank [Microverse](https://www.microverse.org/?grsf=martin-9isn0t) for guiding me through this project, giving code reviews and a community of fellow coders.
+- [ ] I would like to thank [Microverse](https://www.microverse.org/?grsf=martin-9isn0t) for guiding me through this project, giving code reviews and a community of fellow coders.
+
+- [ ] Many thanks to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) for providing the [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) for the app
```

Comment:
- [OPTIONAL] Something weird happened while I was reviewing although the acknowledgment to the author is added in the code I couldn't see it in the browser 👇🏽 Please 🙏🏽 ensure that it appears correctly in the browser.

![image](https://github.com/martinkarugaba/budget_app/assets/58642949/831bb76c-368b-4aca-8c9e-786cb124f2d0)